leetcode/views/result.py

```diff
@@ -102,6 +106,51 @@ def make_compile_error_view(self):
         ]
         return urwid.Padding(urwid.ListBox(urwid.SimpleListWalker(list_items)), left=2, right=2)
 
+    def make_runtime_error_view(self):
+        blank = urwid.Divider()
+        status_header = urwid.AttrWrap(urwid.Text('Run Code Status: '), 'body')
+        status = urwid.AttrWrap(urwid.Text('Runtime Error'), 'hometag')
+        columns = urwid.Columns([(17, status_header), (20, status)])
+        column_wrap = urwid.WidgetWrap(columns)
+        result_header = urwid.Text('--- Run Code Result: ---', align='center')
+        your_input_header = urwid.Text('Last input:')
+        your_input = urwid.Text(self.result['last_testcase'])
+        your_answer_header = urwid.Text('Error:')
```

Comment:
We'd better use the same title as leetcode display. And runtime error should appear before test cases?

Reply:
Updated UI
Leetcode:
![image](https://user-images.githubusercontent.com/1173734/29901004-524dcb96-8daa-11e7-8654-1218c7418317.png)
Terminal:
![image](https://user-images.githubusercontent.com/1173734/29901011-5d1abeb2-8daa-11e7-883c-59c0026c5bac.png)